assets/js/source_lv_hooks.js

```diff
@@ -23,30 +23,22 @@ hooks.SourceLogsSearchList = {
 
     window.scrollTo(0, document.body.scrollHeight)
 
-    window.addEventListener("scroll", _.throttle(resetScrollTracker, 250))
-
-    function resetScrollTracker() {
-      let window_inner_height = window.innerHeight
-      let window_offset = window.pageYOffset
-      let subhead = document.querySelector("div.subhead")
-      let header = document.querySelector("nav")
-      let nav_height = subhead.offsetHeight + header.offsetHeight
-      let client_height = document.body.clientHeight
-
-      // even if we're close to the bottom, we're at the bottom (for mobile browsers)
-      // should make this 40 dynamic
-      if (window_inner_height + window_offset - nav_height >= client_height - 175) {
-        if (window.playing != true) {
-          window.removeEventListener("scroll", _.throttle(resetScrollTracker, 250))
-          hook.pushEvent("soft_play", {})
-          window.playing = true
-        }
-      } else {
-        // We don't reset window.playing here because we don't want it to start playing again when you scroll back down
-        window.removeEventListener("scroll", _.throttle(resetScrollTracker, 250))
-        hook.pushEvent("soft_pause", {})
-      }
-    };
+    const observer =
+      new IntersectionObserver((entries, observer) => {
+        entries.forEach((entry) => {
+          let searchInView = entry.isIntersecting
+          if (searchInView) {
```

Comment:
we can cut out the commented part and use
```
if (!searchInView){
  hook.pushEvent("soft_pause", {})
}
```

Reply:
I think the comment is useful here because I've thought about auto-playing the search when you scroll to the bottom.

Ultimately, it's not what we want to do because you may be looking at some logs which you don't want to lose view of and that could happen if we auto-play.  This comment reminds me that we've tried this before, but it's also useful to uncomment and play with it if you're working on it.